src/UnityMvvmToolkit.UnityPackage/Assets/Plugins/UnityMvvmToolkit/Runtime/UGUI/BindableUGUIElements/BindableDropdown.cs

```diff
@@ -18,37 +18,37 @@ namespace UnityMvvmToolkit.UGUI.BindableUGUIElements
     public class BindableDropdown : MonoBehaviour, IBindableElement
     {
         [SerializeField] private TMP_Dropdown _dropdown;
-        [SerializeField] private string _bindingValuePath;
-        [SerializeField] private string _bindingChoicesPath;
+        [SerializeField] private string _bindingSelectedItemPath;
+        [SerializeField] private string _bindingItemsSourcePath;
 
-        private IProperty<string> _valueProperty;
+        private IProperty<string> _selectedItemProperty;
         private IReadOnlyProperty<ObservableCollection<string>> _itemsSource;
 
-        private PropertyBindingData _propertyBindingData;
+        private PropertyBindingData _selectedItemBindingData;
         private PropertyBindingData _itemsSourceBindingData;
         
         public void SetBindingContext(IBindingContext context, IObjectProvider objectProvider)
         {
-            if (string.IsNullOrWhiteSpace(_bindingChoicesPath))
+            if (string.IsNullOrWhiteSpace(_bindingItemsSourcePath))
```

Comment:
Why do you check only one property? This control, as well as the UI Toolkit version, will throw an exception, if `_bindingSelectedItemPath` is not specified.

The better approach is to check and set them separately.

```csharp
if (string.IsNullOrWhiteSpace(_bindingItemsSourcePath) == false)
{
    // Get _itemsSource.
}

if (string.IsNullOrWhiteSpace(_bindingSelectedItemPath) == false)
{
    // Get _selectedItemProperty.
}
```

And the same logic for `ResetBindingContext` method.

```csharp
if (_itemsSource != null)
{
    // Reset and return to pool.
}

if (_selectedItemProperty != null)
{
    // Reset and return to pool.
}
```